lms/djangoapps/shoppingcart/models.py

```diff
@@ -340,7 +340,6 @@ def generate_registration_codes_csv(self, orderitems, site_name):
             course_id = item.course_id
             course = get_course_by_id(item.course_id, depth=0)
             registration_codes = CourseRegistrationCode.objects.filter(course_id=course_id, order=self)
-            course_info.append((course.display_name, ' (' + course.start_datetime_text() + '-' + course.end_datetime_text() + ')'))
```

Comment:
Looks like the downstream code in `send_confirmation_emails` expects `course_info` to be a list of tuples.